src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function isValidFilm(filmProperty: string, query: string): boolean {
+  return filmProperty.toLowerCase().includes(query.toLowerCase().trim());
+}
+
 export const App: React.FC = () => {
+  const [query, changeQuery] = useState('');
+
```

Comment:
In this case, it is better to use a name that begins with the prefix 'set'. This will allow you to have an understanding of Hook's work. (changeQuery => setQury)